cmd/security-spire-config/seed_builtin_entries.sh

```diff
@@ -25,12 +25,18 @@ echo "local_agent_svid=${local_agent_svid}"
 echo "SPIFFE_SERVER_SOCKET=${SPIFFE_SERVER_SOCKET}"
 echo "SPIFFE_EDGEX_SVID_BASE=${SPIFFE_EDGEX_SVID_BASE}"
 
+echo "SPIFFE_CUSTOM_SERVICES=${SPIFFE_CUSTOM_SERVICES}"
+
+SPIFFE_SERVICES='security-spiffe-token-provider support-scheduler support-notifications
+                     device-bacnet device-camera device-grove device-modbus device-mqtt device-rest device-snmp
+                     device-virtual device-rfid-llrp device-coap device-gpio
+                     app-service-http-export app-service-mqtt-export app-service-sample app-rfid-llrp-inventory
+                     app-service-external-mqtt-trigger'
+
+SEED_SERVICES="${SPIFFE_SERVICES} ${SPIFFE_CUSTOM_SERVICES}"
+
 # add pre-authorized services into spire server entry
-for dockerservice in security-spiffe-token-provider support-notifications support-scheduler \
-    device-bacnet device-camera device-grove device-modbus device-mqtt device-rest device-snmp \
-    device-virtual device-rfid-llrp device-coap device-gpio \
-    app-http-export app-mqtt-export app-sample app-rfid-llrp-inventory \
-    app-external-mqtt-trigger; do
+for dockerservice in $SEED_SERVICES; do
```

Comment:
```suggestion
for dockerservice in "$SEED_SERVICES"; do
```

Reply:
No.  That's not a right suggestion.  We NEED the shell work-breaking on this variable.  No quotes.

Reply:
got cha. ignore my suggestion

Reply:
this didn't work for me - adding the "" means that $SEED_SERVICES isn't unpacked...which breaks the script.